tests/integration_tests/functional/test_api.py

```diff
@@ -795,10 +795,6 @@ def test_send_ctrl_alt_del(test_microvm_with_api):
 
 def _drive_patch(test_microvm):
     """Exercise drive patch test scenarios."""
-    # Patches without mandatory fields are not allowed.
```

Comment:
like mentioned above, we probably still want this check for virtio block devices

Reply:
Added this test back with adjusted error message